model_analyzer/plots/detailed_plot.py

```diff
@@ -121,11 +121,16 @@ def add_run_config_measurement(self, run_config_measurement):
             ['concurrency-range'])
 
         self._data['perf_throughput'].append(
-            run_config_measurement.get_metric_value(tag='perf_throughput'))
+            run_config_measurement.get_non_gpu_metric_value(
+                tag='perf_throughput'))
 
         for metric in self.detailed_metrics:
-            self._data[metric].append(
-                run_config_measurement.get_metric_value(tag=metric))
+            if MetricsManager.is_gpu_metric(tag=metric):
```

Comment:
So the old function `get_metric_value` seemed to handle both gpu and non-gpu cases. Now we are splitting it out and making the user know which it needs to call. Is the reason because one returns a single value and the other returns a list?

Reply:
That's correct.

Reply:
I'm curious how the functions will be used later on. It would be ideal if all of the gpu vs non-gpu information was still hidden from the user so they didn't have to have that knowledge. We can cross that bridge when you make the changes to support multi-model, but maybe get_metric_value returns with an optional "model" arg that returns the single scalar result of the given tag from that model?